src/api/types/general.ts

```diff
@@ -0,0 +1,10 @@
+type Mappings = {
+  domain: string
+  port: string
+  ip: string
+  id: string
+}
+
+type DB = {
+  [mappings: string]: Array<Mappings>
+}
```

Comment:
you should be exporting the type, so you can use it! look at this: https://github.com/garageScript/myProxy/blob/master/src/api/types/admin.ts